packages/schematics/angular/migrations/update-8/index_spec.ts

```diff
@@ -0,0 +1,75 @@
+/**
+ * @license
+ * Copyright Google Inc. All Rights Reserved.
+ *
+ * Use of this source code is governed by an MIT-style license that can be
+ * found in the LICENSE file at https://angular.io/license
+ */
+// tslint:disable:no-big-function
+import { EmptyTree } from '@angular-devkit/schematics';
+import { SchematicTestRunner, UnitTestTree } from '@angular-devkit/schematics/testing';
+
+const renames = [
+  'use-lifecycle-interface',
+  'no-host-metadata-property',
+  'no-outputs-metadata-property',
+  'no-inputs-metadata-property',
+];
+
+describe('Migration to version 8', () => {
+  const schematicRunner = new SchematicTestRunner(
+    'migrations',
+    require.resolve('../migration-collection.json'),
+  );
+
+  let tree: UnitTestTree;
+  const tslintPath = '/tslint.json';
+  const packageJsonPath = '/package.json';
+  const baseConfig = {};
+  const defaultOptions = {};
+  const configPath = `/angular.json`;
+  const tslintConfig = {
+    rules: {
+      'directive-selector': [true, 'attribute', 'app', 'camelCase'],
+      'component-selector': [true, 'element', 'app', 'kebab-case'],
+      'no-output-on-prefix': true,
+      'use-input-property-decorator': true,
+      'use-output-property-decorator': true,
+      'use-host-property-decorator': true,
+      'no-input-rename': true,
+      'no-output-rename': true,
+      'use-life-cycle-interface': true,
+      'use-pipe-transform-interface': true,
+      'component-class-suffix': true,
+      'directive-class-suffix': true,
+    },
+  };
+  const packageJson = {
+    devDependencies: {
+      codelyzer: '^4.5.0',
+    },
+  };
+
+  describe('tslint.json', () => {
```

Comment:
Optional nit: change the description of this test, and you are also testing that the `package.json`  is updated.